src/kernels/conv1x1u.s

```diff
@@ -288,6 +288,9 @@ raw_filter_dword_k_cnt = 1
     .VGPR_ALLOC tid
     .VGPR_ALLOC voffset_in
     .VGPR_ALLOC voffset_out
+    .if (.amdgcn.gfx_generation_number == 9 && .amdgcn.gfx_generation_stepping == 10)
+    .VGPR_ALLOC vtmp_align, (.VGPR_NEXT_FREE % 4) // buffer_load_dwordx4 vdst must be 64 bit aligned
```

Comment:
Do you mean that `inputA/B` must be quad-aligned? Please fix or remove the comment: 4x32 = 128 bit.

Is it so that only gfx90a requires this alignment?

[Recommendation] `vtmp_align` -> `valign_unused` or something else like this.

[Note] Maybe we should update .VGPR_ALLOC: add an optional parameter there, like it is done in ".SGPR_ALLOC", but this can be posponed.

Reply:
@Slimakanzer Wow. So I managed to find a bug?